src/harness/sas_testcase.py

```diff
@@ -96,13 +96,42 @@ def assertRegisteredAndGranted(self, registration_request, grant_request,
       self.assertEqual(grant_resp['response']['responseCode'], 0)
 
       list_of_cbsd_ids_and_grant_ids.append({
-          'cbsdId': grant_resp['cbsdId'],
-          'grantId': grant_resp['grantId']
+        'cbsdId': grant_resp['cbsdId'],
+        'grantId': grant_resp['grantId']
       })
 
     # Return cbsd_ids and grant_ids
     return list_of_cbsd_ids_and_grant_ids
 
+  def assertHeartbeatsSuccessful(self, cbsd_ids, grant_ids, operation_states):
```

Comment:
It probably makes sense to have the output format of assertRegisteredAndGranted() match the input format of assertHeartbeatsSuccessful(). I'm not sure which should change though. Thoughts? (My gut reaction is that assertRegisteredAndGranted() should change, since it's the oddball right now.)